certify-core/src/main/java/io/mosip/certify/core/dto/OIDCTransaction.java

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
What is the dependency on OIDCTransaction?

Reply:
removed this